src/hooks/WeatherHook/index.tsx

```diff
@@ -2,20 +2,22 @@ import React, { createContext, useCallback, useState, useContext } from 'react';
 import { climateCityDetails } from '../../services/api.requests';
 import { ClimateDetails } from '../../services/services.interface';
 import toFahrenheit from '../../utils/functions/toFahrenheit';
+import { tempUnitToken } from '../../store';
 
 interface WeatherContextData {
 	climate: ClimateDetails | null;
 	tempUnit: 'celcius' | 'fahrenheit';
 	getClimate(woeid: number): void;
 	changeTempUnit(unit: WeatherContextData['tempUnit']): void;
+	setClimate: React.Dispatch<React.SetStateAction<ClimateDetails | null>>;
```

Comment:
i know, this is not too smart, but it's a workaround, just for now. We can discuss about it.